Installer fixes

Stanzilla opened the pull request:
* fix the finish page so it launches KVIrc as the user that ran the install before UAC privilege escalation.
* remove autostart related functions
* remove Winamp installer option


Rebase of https://github.com/kvirc/KVIrc/pull/2147 + additions from https://github.com/kvirc/KVIrc/issues/1691

Stanzilla committed:
Fixed the finish page so it will launch KVIrc as the user that ran the install before UAC privilege escalation.

Rebase of https://github.com/kvirc/KVIrc/pull/2147

Dessa commented:
here is hoping that the plugin actually works with nsis3. if not there is also https://github.com/lordmulder/stdutils but that does only work with 32 bit nsis (and if it works in nsis3 then it is "by luck")

Stanzilla committed:
remove autostart related functions

autostart was removed as an option in https://github.com/kvirc/KVIrc/commit/27f5d68ddf1b1bc4473b569f936ebd6db298977e#diff-b5bbda0fcc3b9962928ec48d87070d82

Stanzilla committed:
Remove Winamp installer option

as per https://github.com/kvirc/KVIrc/issues/1691

Stanzilla commented:
stdutils would also help the new items in https://github.com/kvirc/KVIrc/issues/1691 taskbar pinning etc https://stackoverflow.com/questions/24326685/pin-icons-to-taskbar?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Stanzilla committed:
remove unused CheckUserInstallRights function

Stanzilla committed:
don't use the NSIS plugin

* the explorer.exe way actually works and does not need extra code
* remove no longer needed UserInfo.dll
* clean stale onInstSuccess function
